bin/mysociety

```diff
@@ -121,12 +125,26 @@ case $COMMAND in
         COMMAND=deploy
         VHOST=$1
         shift || die "specify a command or a virtual host"
-        if [ "$VHOST" == "stop" -o "$VHOST" == "update" -o "$VHOST" == "start" -o "$VHOST" == "remove" -o "$VHOST" == "servers" -o "$VHOST" == "balancers" ]
+        if [ "$VHOST" == "stop" -o "$VHOST" == "update" -o "$VHOST" == "start" -o "$VHOST" == "remove" -o "$VHOST" == "servers" -o "$VHOST" == "balancers" -o "$VHOST" == "package" ]
         then
             COMMAND=$VHOST
             VHOST=$1
             shift || die "specify a virtual host"
         fi
+        if [[ "$COMMAND" == "package" || $* == *"--from-package"* ]]
+        then
+            if [ -z "$VHOST_PACKAGE" ]
+            then
+                # Assuming bookworm only for now.
+                VHOST_PACKAGE=$(ssh msdevpkg003cama "sudo su vpackager -c 'vpackager build --vhost $VHOST  --debian-version bookworm  --vhosts-config /data/vhosts.json --output-dir /data/packages'")
```

Comment:
Couple of thoughts here, although I appreciate we're at the MVP stage.

- I would suggest using a CNAME here (as in `deploy-vhost`); although as we're using SSH we'd need to manage key rotations when server-side keys change. I think ultimately doing this as an HTTP POST would be a good solution, as one of our ultimate goals is to stop doing all this over SSH.

- Perhaps we could do `PKG_DEBIAN_VER=${PKG_DEBIAN_VER:-bookworm}` somewhere then have `--debian-version ${PKG_DEBIAN_VER}` so we only need to update this script when we want to change the default Debian version?

Reply:
Ticket for HTTP approach, in case you want to add any more detail: https://github.com/mysociety/sysadmin/issues/2116